libraries/fc/src/log/console_appender.cpp

```diff
@@ -20,7 +20,7 @@ namespace fc {
    public:
      config                      cfg;
      color::type                 lc[log_level::off+1];
-     bool                        use_syslog_header{getenv("JOURNAL_STREAM")};
+     bool                        use_syslog_header{static_cast<bool>(getenv("JOURNAL_STREAM"))};

```

Comment:
I think it would be better as:
`bool                        use_syslog_header{getenv("JOURNAL_STREAM") != nullptr};`

Reply:
I agree; made the change.